frontend/src/metabase/ui/components/buttons/Button/Button.module.css

```diff
@@ -81,6 +81,42 @@
     }
   }
 
+  &[data-variant="visualizer"] {
+    --button-color: var(--mb-color-brand);
+    --button-color-hover: var(--mb-color-brand-light);
```

Comment:
`metabase/ui` components must be generic and should not be tied to a specific product area or use case. If we're planning to use this variant outside the visualizer, we should give it a more generic name. If not, I'd suggest we add a new button component to `metabase/visualizer`. WDYT?